lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -137,7 +137,7 @@ def self.create(lookup_invocation, config_path, owner)
           if parsed.is_a?(Hash)
             parsed
           else
-            Puppet.warning("#{config_path}: File exists but does not contain a valid YAML hash. Falling back to Hiera version 3 default config")
+            Puppet.warning(_("%{config_path}: File exists but does not contain a valid YAML hash. Falling back to Hiera version 3 default config") % { config_path: config_path })
```

Comment:
@Magisus @MosesMendoza YAML is a keyword referring to a language, should it be left out of translation?